config/formats.ts

```diff
@@ -879,6 +879,132 @@ export const Formats: FormatList = [
 			}
 		},
 	},
+	{
+		name: "[Gen 9] Fortemons",
+		desc: `Put a move in the item slot to have all of a Pok&eacute;mon's moves inherit its properties.`,
+		threads: [
+			`&bullet; <a href="https://www.smogon.com/forums/threads/3713983/">Fortemons</a>`,
+		],
+
+		mod: 'gen9',
+		searchShow: false,
+		ruleset: ['Standard OMs', 'Sleep Clause Mod', 'Min Source Gen = 9'],
+		banlist: ['Koraidon', 'Miraidon', 'Palafin', 'Covert Cloak', 'Fake Out'],
+		restricted: ['Dynamic Punch', 'Inferno', 'Mud Slap', 'Nuzzle', 'Power Trip', 'Rapid Spin', 'Stored Power', 'Zap Cannon'],
+		validateSet(set, teamHas) {
+			const item = set.item;
+			const species = this.dex.species.get(set.species);
+			const move = this.dex.moves.get(item);
+			if (!move.exists || move.category === 'Status' || this.ruleTable.isRestricted(`move:${move.id}`) ||
+				move.flags['charge'] || move.priority > 0 || move.damageCallback) {
+				return this.validateSet(set, teamHas);
+			}
+			set.item = '';
+			const problems = this.validateSet(set, teamHas) || [];
+			set.item = item;
+			if (this.checkCanLearn(move, species, this.allSources(species), set)) {
+				problems.push(`${species.name} can't learn ${move.name}.`);
+			}
+			if ((move.secondaries?.some(secondary => secondary.boosts?.accuracy && secondary.boosts.accuracy < 0) ||
+				move.multihit || move.id === 'beatup') &&
+				!this.ruleTable.has(`+move:${move.id}`)) {
+				problems.push(`${move.name} can't be used as an item.`);
```

Comment:
```suggestion
				problems.push(`The move ${move.name} can't be used as an item.`);
```
this is clearer, i think